index.html

```diff
@@ -9,18 +9,23 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>My Portfolio</title>
     <header>
+        <ul class="nav-menu">
```

Comment:
I think you have to remove your code out of the header tag i.e. head must wrap only tags such as title, encoding, scripts and the like